### PR TITLE
DOC-7624: Update Activate diagnostics collection with warning

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses: [DOC-7624](https://cockroachlabs.atlassian.net/browse/DOC-7624)

### CODE REVIEW

File: _includes/v23.1/ui/statement-details.md [hunk 1]
```diff
@@ -105,17 +105,20 @@ Diagnostics will be collected a maximum of *N* times for a given activated finge
 
 <img src="{{ 'images/v23.1/ui_activate_diagnostics.png' | relative_url }}" alt="Activate statement diagnostics" style="border:1px solid #eee;max-width:100%" />
 
+{{site.data.alerts.callout_danger}}Collecting diagnostics has an impact on performance. All executions of the statement fingerprint will run slower until diagnostics are collected.
```
Comment (gemma-shay) [inline _includes/v23.1/ui/statement-details.md:108]:
```suggestion
{{site.data.alerts.callout_danger}}
Collecting diagnostics has an impact on performance. All executions of the statement fingerprint will run slower until diagnostics are collected.
```

Comment (florence-crl) [reply _includes/v23.1/ui/statement-details.md]: updated file with change

File: _includes/v23.1/ui/statement-details.md [hunk 2]
```diff
@@ -105,17 +105,21 @@ Diagnostics will be collected a maximum of *N* times for a given activated finge
...
+
 To activate diagnostics collection:
 
 1. Click the **Activate diagnostics** button. The **Activate statement diagnostics** dialog displays.
 
-    <img src="{{ 'images/v23.1/ui_activate_diagnostics_dialog.png' | relative_url }}" alt="Statements diagnostics dialog" style="border:1px solid #eee;max-width:100%" />
-
-1. Choose whether to activate collection on the next statement execution (default) or if execution latency exceeds a certain time. If you choose the latter, accept the default latency of 100 milliseconds, or specify a different time. All executions of the statement fingerprint will run slower until diagnostics are collected.
-1. Choose whether the request should expire after 15 minutes, or after a different the time, or disable automatic expiration by deselecting the checkbox.
+1. Choose whether to
```
Comment (gemma-shay) [inline _includes/v23.1/ui/statement-details.md:116]:
```suggestion
1. Choose whether to:
```

**Review by gemma-shay — APPROVED** [review]
small nit

**Pull request merged**